src/NzbDrone.Core/Indexers/Newznab/NewznabCapabilitiesProvider.cs

```diff
@@ -73,6 +74,13 @@ private NewznabCapabilities FetchCapabilities(NewznabSettings indexerSettings)
                 _logger.Debug(ex, "Failed to parse newznab api capabilities for {0}", indexerSettings.BaseUrl);
                 throw;
             }
+            catch (ApiKeyException ex)
+            {
+                ex.WithData(response, 128 * 1024);
+                _logger.Trace("Unexpected Response content ({0} bytes): {1}", response.ResponseData.Length, response.Content);
+                _logger.Debug(ex, "Failed to parse newznab api capabilities for {0}, invalid key", indexerSettings.BaseUrl);
```

Comment:
```suggestion
                _logger.Debug(ex, "Failed to parse newznab api capabilities for {0}, invalid API key", indexerSettings.BaseUrl);
```

to more closely match the UI?